ftl/core/deck-config.ftl

```diff
@@ -560,6 +560,10 @@ deck-config-compute-optimal-retention-tooltip =
     if it significantly differs from 0.9, it's a sign that the time you've allocated each day is either too low
     or too high for the amount of cards you're trying to learn. This number can be useful as a reference, but it
     is not recommended to copy it into the desired retention field.
+deck-config-health-check-tooltip =
+    This will show a warning if FSRS struggles to adapt to your memory.
+
+    Health check is performed only when using Optimize Current Preset.
```

Comment:
It might be worth splitting the second sentence into a separate string and joining them together, so that we don't break the first translation if we add the health check to 'optimize all'